src/utils/dom-utils/viewOnOpenSauced.ts

```diff
@@ -8,12 +8,12 @@ const injectViewOnOpenSaucedButton = (username: string) => {
 
     const viewOnOpenSaucedButton = ViewOnOpenSaucedButton(username);
 
-    const user = document.getElementsByClassName(
+    const userEditableArea = document.getElementsByClassName(
         GITHUB_PROFILE_USER_PROFILE_EDITABLE_AREA_SELECTOR,
     );
-    const userDef = user[0] as HTMLElement;
+    const editableAreaElement = userEditableArea[0] as HTMLElement;
```

Comment:
No need to cast. The `Element` type has a `parentNode`. See this [TS Playground example](https://www.typescriptlang.org/play?#code/MYewdgzgLgBA4gSQCoAkCqAhA+gBQEoDyAYggDICiWaAyuXroSRVuQCLICCGzHe5HWWhQDCSAnhgBeGAHIIIALYBTGBCUAbJcCggATjICwAKFCRYANwCWSgO4EwBAA5Kw1AIYBXYEoAmGD1A6YFIwPiDAHspgUAB0wLpKblBK5JpRUAAUMj6W5jIAlMam0DAearrkOVBuAEaaHAluIWER6TEA5kpQqUrpEBgAnsLqbhAQAHJuyhnGMHPwyOjY+MRklDR0DKvMbJzclLz8guQiYngANMb5ANxF4CW+ltV1Sg2JPekhZUoVVbX1jQA2gAGAC6xmMj2eAPeaRcsUcbgS0XGIB8SgA-DFLJAflAMEoAGZ6JQZKy2exOFzuLy+fyBcDnGBQ-6vRofeE3IA)

```suggestion
    const editableAreaElement = userEditableArea[0];
```